src/job.js

```diff
@@ -0,0 +1,112 @@
+import EventObserver from './events';
+import { protobuf, unserialize } from './protobuf';
+
+export default class Job {
+  constructor({ worker, modelId, modelVersion, gridClient, logger }) {
+    this.worker = worker;
+    this.modelId = modelId;
+    this.modelVersion = modelVersion;
+    this.grid = gridClient;
+    this.plans = {};
+    this.protocols = {};
+    this.cycleConfig = {};

```

Comment:
What is this? As a worker you won't have the `server_config` sent to you (if that's what you're referencing).

Reply:
It's params from cycle request response